common/deployment/src/main/java/io/quarkus/amazon/common/deployment/DevServicesLocalStackProcessor.java

```diff
@@ -304,6 +305,10 @@ public String getSecretKey() {
                                         .toArray(EnabledService[]::new))
                                 .withLabel(DEV_SERVICE_LABEL, devServiceName);
 
+                        if (localStackDevServicesBuildTimeConfig.accessToHost()) {
+                            Testcontainers.exposeHostPorts(localStackDevServicesBuildTimeConfig.hostPort());
+                        }
```

Comment:
```suggestion
                        container.withAccessToHost(localStackDevServicesBuildTimeConfig.accessToHost);
```

This would force the host access setup and let users configure exposed host port themself.

Reply:
How would users be able to configure exposed port?
I've tried by intercepting the quarkus startup event on my application:
```
@ApplicationScoped
@IfBuildProfile("dev")
public class DevLifecycleBean {

    @ConfigProperty(name = "quarkus.http.port")
    int port;

    void onStart(@Observes StartupEvent ev) {
        Testcontainers.exposeHostPorts(port);
    }
}
```
`exposeHostPorts()` gets called with the right port as argument but then connection to host is refused, I've not investigated why though.

Apart from the difficulties, I do not like much this solution because a seamless integration is being traded for boilerplate code + testcontainers runtime dependency on the application side. 
In my solution the host port is user-configurable via `host-port` property, however most of the time would be retrieved from `quarkus.http.port` property without requiring tweaks.
An edit I could make is accepting multiple ports instead of just 1, if that's useful.
The same approach is used on the Quarkus Jib extension.

https://github.com/quarkusio/quarkus/blob/0f776ad0434619b428439c1d97257cf1555dd76c/extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/ContainerImageJibConfig.java#L116-L120